### PR TITLE
[rabbitmq-ha] AppVersion to 3.7.4 and more configuration parameters added

### PR DESCRIPTION
**Pull request by PetteriVaarala**
**What this PR does / why we need it**:
- Latest RabbitMQ version to Chart.yml
- New configuration parameters
    - customSecret similar to customConfigMap
    - terminationGracePeriodSeconds
    - Name for persistentVolume
- Use rabbitmqManagerPort value in statefulset container port instead of hardcoded value

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: -

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/rabbitmq-ha/templates/statefulset.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ spec:
               containerPort: 5672
             - name: http
               protocol: TCP
-              containerPort: 15672
+              containerPort: {{ .Values.rabbitmqManagerPort }}
```
Comment (rjkernick) [inline stable/rabbitmq-ha/templates/statefulset.yaml:51]: Please add to values.yaml

Comment (rjkernick) [reply stable/rabbitmq-ha/templates/statefulset.yaml:51]: Also add default value to readme

File: stable/rabbitmq-ha/values.yaml [hunk 2]
```diff
@@ -189,6 +192,7 @@ persistentVolume:
   ##   GKE, AWS & OpenStack)
   ##
   # storageClass: "-"
+  name: data
```
Comment (rjkernick) [inline stable/rabbitmq-ha/values.yaml:195]: Please add to readme

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
Please rebase master and fix merge conflicts

File: stable/rabbitmq-ha/templates/statefulset.yaml (hunk 1 above)
Comment (PetteriVaarala) [reply stable/rabbitmq-ha/templates/statefulset.yaml:51]: It already is in both values.yaml and readme, just was not used here.

File: stable/rabbitmq-ha/values.yaml (hunk 2 above)
Comment (PetteriVaarala) [reply stable/rabbitmq-ha/values.yaml:195]: Added to readme

File: stable/rabbitmq-ha/templates/statefulset.yaml (hunk 1 above)
Comment (rjkernick) [reply stable/rabbitmq-ha/templates/statefulset.yaml:51]: Thanks!

**Review by rjkernick — APPROVED** [review]

File: stable/rabbitmq-ha/templates/statefulset.yaml (hunk 1 above)
Comment (francesco-beccaria) [reply stable/rabbitmq-ha/templates/statefulset.yaml:51]: This change seems to break things: #5577

**Pull request merged**